example_data/sample_bundle.json

```diff
@@ -108,45 +126,54 @@
             {
               "system": "http://snomed.info/sct",
               "code": "122555007",
-              "display": "Venus blood specimen"
+              "display": "blood"
             }
           ]
         },
         "subject": {
           "type": "Patient",
-          "reference": "Patient/37176e84-d977-e993-3c49-d76fcfc6e625"
+          "reference": "Patient?_id=gosh-redcap-40388914"
         }
       },
       "request": {
-        "method": "POST",
-        "url": "Specimen"
+        "method": "PUT",
+        "url": "Specimen?_id=specimen-20RG-205G0047"
       }
     },
     {
-      "fullUrl": "Practitioner/d3447490-96fd-35d0-adf2-0806e5214606",
+      "fullUrl": "Practitioner/practitioner1-20RG-205G0047",
       "resource": {
         "resourceType": "Practitioner",
-        "id": "d3447490-96fd-35d0-adf2-0806e5214606",
+        "id": "practitioner1-20RG-205G0047",
+        "identifier": [
+          {
+            "value": "practitioner1-20RG-205G0047"
+          }
+        ],
         "active": true,
         "name": [
           {
             "use": "official",
-            "family": "Nephrology",
+            "family": "Taylor-Beadling",
             "given": [
-              "Clinics"
+              "Alison"
             ],
             "text": "Authorized By"
           }
         ],
         "qualification": [
           {
             "code": {
-              "text": "Principal Clinical Scientist"
+              "coding": [
+                {
+                  "display": "Principal Clinical Scientist"
+                }
+              ]
```

Comment:
We've changed this from coding.text, is there a reason why? Feels like text makes the most sense here?

Reply:
Ok.